src/assets/stack-blitz/src/index.html

```diff
@@ -1,5 +1,13 @@
-<link href="https://fonts.googleapis.com/icon?family=Material+Icons&display=block" rel="stylesheet">
-<div class="mat-app-background basic-container">
-  <material-docs-example>loading</material-docs-example>
-</div>
-<span class="version-info">Current build: {{version}}</span>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
```

Comment:
```suggestion
  <meta charset="utf-8">
```
For consistency with what's generated via `ng new`.  This value is case-insensitive .